python/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='glowpy',
+    version='0.1.0',
+    packages=['glow'],
+    install_requires=[
+        'typeguard==2.5.0',
+    ],
+    author='Glow Project',
+    description='Glow: Genomics on Apache Spark',
```

Comment:
Should use same messaging as website: "An open-source toolkit for large-scale genomic analysis"